apps/api/.env.test

```diff
@@ -2,7 +2,7 @@
 PORT=3000
 
 # DB
-PG_USER=postgres
-PG_PASSWORD=mypassword
-PG_HOST=localhost:5432
+PG_USER=testUser
+PG_PASSWORD=testPassword
 PG_DATABASE=postgres
+PG_HOST=localhost:5000
```

Comment:
Just differentiating from .env.schema values

Reply:
On second review, that example may not work with our tests, since the env might be set by the `before()` function after the env is read by the startDb function... Need to test and confirm, I have ideas if need be. I stand by the idea that we shouldn't be feeding an env file into our test suite except to configure/control what tests are running or how they are being run.

Reply:
Setting process.env does work but it's not actually necessary because we can just set the container variables and obtain the connection string like so:

```
container = await new PostgreSqlContainer()
			.withUsername(PG_USER)
			.withPassword(PG_PASSWORD)
			.withDatabase(PG_DATABASE)
			.start();

const connectionString = container.getConnectionUri();
db = connectToDb(connectionString);
```

where PG_USER etc. are defined as constants in a test file